test/extended/fixtures/ldap/ldapserver-buildconfig.json

```diff
@@ -17,6 +17,7 @@
       },
       "contextDir": "images/openldap"
     },
+    "runPolicy": "Serial",
```

Comment:
hmm good question, I have to try to create BC without that field set. I think the defaulter should default to "Serial" in case it is not set, but for some reason, the test using this file started to fail....

actually, I think this might be related to the defaulters changed... I will look at this.